frontend/vue/components/CodeExercise/CodeArea.vue

```diff
@@ -9,6 +9,7 @@ import { EditorView, keymap } from '@codemirror/view'
 import { EditorState, StateField } from '@codemirror/state'
 import { defaultKeymap, indentWithTab } from '@codemirror/commands'
 import { python } from '@codemirror/lang-python'
+import { View } from 'backbone'
```

Comment:
i am not sure why this was added, i do not see it being used anywhere in this file... maybe it was a mistake?